src/rules/max-line-length/README.md

```diff
@@ -74,3 +74,37 @@ a { color: pink; background: orange; }
  */
 a { color: pink; /* this comment is also long but not on its own line */ }
 ```
+
+### `ignore: ["comments"]`
+
+Only enforce the line-length limit for lines that are not commented out.
+
+This also applies to comments that are stuck in between other stuff.
```

Comment:
Maybe then - "stuck in between a css code. It describes the following example: `a { color: /*